backend/src/main/java/com/capstone/moneytree/service/impl/DefaultTransactionService.java

```diff
@@ -126,6 +131,20 @@ private void executeTransaction(String alpacaKey, Order order, User user) {
       }
    }
 
+   private Order executeRequest(AlpacaAPI api, Order order) throws AlpacaAPIRequestException {
+      if (MoneyTreeOrderType.MARKET_BUY.name().equals(obtainMoneyTreeOrderType(order).name()) ||
```

Comment:
We'll have to test those two condition paths when the executeTransaction test is fixed.

Reply:
Added extra tests cases